app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,7 @@
   <body>
     <%= render 'shared/header' %>
     <main id="main-content" class="<%= params[:controller] %>">
+      <%= render 'shared/messages' %>
```

Comment:
hmmm this might make it so the other pages have unexpected behavior, such as the login page when erroring out and such

Reply:
Yea, just noticed that `<%= render "shared/messages" %>` was already the `home/index` view. For admin alerts, we could also render the partial in `users/index`, or just have only one call in `layouts/application` for global alerts. Got a preference?